src/main/charts/bitbucket/values.yaml

```diff
@@ -451,7 +451,12 @@ bitbucket:
     # -- The GID used by the Bitbucket docker image
     #
     gid: "2003"
-    
+
+  # -- Boolean to define whether to set home directory permissions on startup
```

Comment:
I think we can add `local` to the description => `to set local home directory`
This will make it more explicit and avoid potential confusion between local or shared home directory.